check_api/src/test/java/com/google/errorprone/ErrorProneOptionsTest.java

```diff
@@ -164,6 +166,20 @@ public void recognizesCompilingTestOnlyCode() {
   }
 
   @Test
+  public void recognizesExcludedPaths() {
```

Comment:
Can you add an integration test that verifies the option is actually used to filter the compilation units that get analyzed?

Reply:
Will look into it; is there an existing test whose structure I can copy?

Reply:
There are test cases in [`ErrorProneJavaCompilerTest`](https://github.com/google/error-prone/blob/a9f4c95bb5648a8e1ef56e6570033ebaa4ad95f1/core/src/test/java/com/google/errorprone/ErrorProneJavaCompilerTest.java) that use flags to enable/disable checks and change severities. Maybe something like that?

Reply:
Cool, thanks!  I added a test case there.